isis/tests/Fixtures.cpp

```diff
@@ -253,7 +255,110 @@ namespace Isis {
     delete threeImageOverlapFile;
     delete twoImageOverlapFile;
   }
-  
+
+  void ApolloNetwork::SetUp() {
+    TempTestingFiles::SetUp();
+    
+    isdFile1 = new FileName("data/apolloNetwork/Sub4-AS15-M-0583_msk.isd");
+    isdFile2 = new FileName("data/apolloNetwork/Sub4-AS15-M-0584_msk.isd");
+    isdFile3 = new FileName("data/apolloNetwork/Sub4-AS15-M-0585_msk.isd");
+    isdFile4 = new FileName("data/apolloNetwork/Sub4-AS15-M-0586_msk.isd");
+    isdFile5 = new FileName("data/apolloNetwork/Sub4-AS15-M-0587_msk.isd");
+    isdFile6 = new FileName("data/apolloNetwork/Sub4-AS15-M-1423.isd");
+    isdFile7 = new FileName("data/apolloNetwork/Sub4-AS15-M-1537.isd");
+
+    label1 = new FileName("data/apolloNetwork/Sub4-AS15-M-0583_msk.pvl");
+    label2 = new FileName("data/apolloNetwork/Sub4-AS15-M-0584_msk.pvl");
+    label3 = new FileName("data/apolloNetwork/Sub4-AS15-M-0585_msk.pvl");
+    label4 = new FileName("data/apolloNetwork/Sub4-AS15-M-0586_msk.pvl");
+    label5 = new FileName("data/apolloNetwork/Sub4-AS15-M-0587_msk.pvl");
+    label6 = new FileName("data/apolloNetwork/Sub4-AS15-M-1423.pvl");
+    label7 = new FileName("data/apolloNetwork/Sub4-AS15-M-1537.pvl");
```

Comment:
probably a dumb question: can we get away with less? If not, can we use an arrays?

Reply:
Someone could pull these up in qmos or qgis and look at the overlap. We want to have a few points with 3-4 measures but not all of them are needed. It's possible some of these could go away. I don't think an array will save any hassle here, each FileName still has to be initialized by itself.

Reply:
If the filenames are also programmatic it would fix that I think (like in my comment at the bottom). Something like labels[i] = new FileName("data/apolloImage" + i + ".pvl");

Reply:
On the "using an array" thing. I expected this to work, and thought it was a good idea, and then when I tried it, I ran into trouble. I nominate this specific thing for being a thing you update in a future PR if you it's not out of the way.